Kiosk/Bid Fulfillment/BidderNetworkModel.swift

```diff
@@ -1,12 +1,11 @@
 import Foundation
 import ReactiveCocoa
-import Swift_RAC_Macros
 import Moya
 
 class BidderNetworkModel: NSObject {
     // MARK: - Getters
 
-    var fulfillmentController: FulfillmentController
+    unowned let fulfillmentController: FulfillmentController
```

Comment:
Makes sense – something to keep in mind when I get to https://github.com/artsy/eidolon/issues/433